pkgs/games/endless-sky/fixes2.patch

```diff
@@ -0,0 +1,40 @@
+diff --git a/source/Files.cpp b/source/Files.cpp
```

Comment:
It seems weird to have two patch files here, could we combine them into one?

Reply:
done